src/Parser/Priest/Discipline/Modules/Spells/TwistOfFate.js

```diff
@@ -60,8 +63,32 @@ class TwistOfFate extends Analyzer {
     this.healing += calculateEffectiveHealing(event, TWIST_OF_FATE_HEALING_INCREASE);
   }
 
+  suggestions(when) {
+    if (this.hasSoulOfTheHighPriest === false) {
+      when(this.owner.getPercentageOfTotalHealingDone(this.healing)).isLessThan(SuggestionThresholds.TWIST_OF_FATE_TALENT_PERCENT.minor)
```

Comment:
This is very long and I don't think moving the suggestion threshold out of this class makes this much easier to maintain, but it does make it harder to find the actual thresholds. In general you fiddle with suggestion thresholds one at a time and having the entire context of the suggestion allows you to be more precise. 

I recommend going for a hardcoded value for the initial threshold and then do regular `recommended - 0.005` and major similar to that.